tests/UnitTests/CoreGraphics.drawing/CGContextDrawingTests.cpp

```diff
@@ -14,30 +14,20 @@
 //
 //******************************************************************************
 
-#import "DrawingTest.h"
-#import <windows.h>
-#import "CGContextInternal.h"
+#include "DrawingTest.h"
+#include "DrawingTestConfig.h"
+#include "ImageHelpers.h"
+#include <windows.h>
 
-// TODO:#1448 Remove the use of OBJC code
 #ifdef WINOBJC
-
-static NSString* getModulePath() {
-    char fullPath[_MAX_PATH];
-    GetModuleFileNameA(NULL, fullPath, _MAX_PATH);
-    return [@(fullPath) stringByDeletingLastPathComponent];
-}
-
-static NSString* getPathToFile(NSString* fileName) {
-    static StrongId<NSString*> refPath = getModulePath();
-    return [refPath stringByAppendingPathComponent:fileName];
-}
+#include "CGContextInternal.h"
 
 DISABLED_DRAW_TEST_F(CGContext, DrawIntoRect, UIKitMimicTest) {
     // Draw a portion of an image into a different region.
-    CFDataRef data = (CFDataRef)[NSData dataWithContentsOfFile:getPathToFile(@"png3.9.png")];
-    woc::unique_cf<CGDataProviderRef> dataProvider(CGDataProviderCreateWithCFData(data));
+    auto drawingConfig = DrawingTestConfig::Get();
```

Comment:
Very true.

---
In reply to: [90145740](https://github.com/Microsoft/WinObjC/pull/1470#discussion_r90145740) [](ancestors = 90145740)